data/settings.js

```diff
@@ -515,123 +515,290 @@ depending on the authenticated user).`
 
 	/* Pigeonhole plugin settings. */
 
```

Comment:
Rather huge commit, so perhaps would be easier to process this by splitting based on some directories, at least data/settings.js change to its own commit.

Reply:
That is why it is wip

Reply:
fixed